app/helpers/user_sessions_helper.rb

```diff
@@ -11,6 +11,10 @@ def user_signed_in?
     !current_user.nil?
   end
 
+  def user_admin?
+    current_user&.admin?
+  end
+
```

Comment:
Ce helper est mal nommé: `user` dans le contexte peut-être le user qu'on regarde, c'est confus.
```suggestion
  def current_user_admin?
    user_signed_in? &&
      current_user.admin?
  end
```

Reply:
Ok pour le naming. Pour l'implem par contre ta proposition me lève cette erreur quand le user est pas connecté 🙃  

```
ActionView::Template::Error:
       undefined method `admin?' for nil:NilClass 
```

Du coup je vais garde `current_user&.admin?`

Reply:
Pour moi faut fixer `user_signed_in?` ici.

Mon implémentation a le mérite d'être plus claire.

Reply:
C'est un stub de `user_signed_in?` dans `spec/views/jwt_api_entreprise/_jwt_api_entreprise_spec.rb` qui lève l'erreur... Je vais migrer ces specs dans des specs de feature.